jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriter.java

```diff
@@ -178,7 +178,7 @@ public void doWrite(Server server, Query query, Iterable<Result> results) throws
 
 			HashMap<String, Object> filteredValues = newHashMap();
 			Object value = result.getValue();
-			if (isValidNumber(value)) {
+			if (isValidNumber(value) || value instanceof String) {
```

Comment:
It seems strange to only allow "valid numbers". InfluxDB for sure [allows to store Strings](https://docs.influxdata.com/influxdb/v1.5/write_protocols/line_protocol_reference/).